com.chartboost.mediation/Editor/Adapters/AdaptersWindowSerialization.cs

```diff
@@ -10,6 +10,9 @@ namespace Chartboost.Editor.Adapters
 {
     public partial class AdaptersWindow
     {
+        /// <summary>
+        /// Loads partner versions & current adapter selections.
```

Comment:
So weird how GitHub is highlighted '&' as red. 🤔

Reply:
I will change it, it's weird though LOL